siliconcompiler/tool.py

```
@@ -1761,62 +1755,66 @@ def _generate_doc(self, doc,
+                        doc.env.docname,
+                        key_text=["...", "var", key])
+
+                param = self.get("var", key, field=None)
+                help_str = param.get(field="help")
+
+                val_type = param.get(field="type")
+                if "<" in val_type:
+                    encode_type = NodeType.parse(val_type)
+                    try:
+                        if val_type.startswith('['):
+                            allowed = list(encode_type)[0].values
```

Ruff (0.13.2)

Prefer `next(iter(encode_type))` over single element slice

Replace with `next(iter(encode_type))`

(RUF015)